FEXCore/include/FEXCore/Utils/StringUtils.h

```diff
@@ -28,4 +28,41 @@ namespace FEXCore::StringUtils {
     return RightTrim(LeftTrim(std::move(String), TrimTokens), TrimTokens);
   }
 
+  namespace StringHasher {
+    struct Hasher {
+      // Jenkins one-at-a-time function.
+      // Based off of https://en.wikipedia.org/wiki/Jenkins_hash_function
+      // citing: https://www.drdobbs.com/database/algorithm-alley/184410284
+      // and citing: https://www.burtleburtle.net/bob/hash/doobs.html
+      //
+      // Good enough constexpr hasher.
+      uint32_t constexpr operator()(const char* str) const {
```

Comment:
```suggestion
      size_t constexpr operator()(const char* str) const noexcept {
```

would make this match the interface requirements of std::hash